code/rdb/datastripe.q

```diff
@@ -27,7 +27,7 @@ initdatastripe:{
     endofperiod::.rdb.datastripeendofperiod;
     .rdb.tablekeycols:.ds.loadtablekeycols[];
     t:tables[`.] except .rdb.ignorelist;
-    .ds.access:([table:t] start:.ds.getstarttime each t; end:0Np ; stptime:0Np ; keycol:`sym^.rdb.tablekeycols[t]);
+    .ds.access:([table:t] start:.ds.getstarttime each t; end:0Np ; stptime:0Np ; keycol:`sym^.rdb.tablekeycols[t]; segment:first .ds.segmentid);
```

Comment:
Any reason why this is first?

Reply:
.ds.segmentid is a one item list, not really sure why, but I imagine this is why Ethan is using first to get the value

Reply:
.ds.segmentid is a list of one symbol, was throwing a `length` error when loading the process code